server/config/config.go

```diff
@@ -21,6 +21,12 @@ const (
 
 	// DefaultGRPCWebAddress defines the default address to bind the gRPC-web server to.
 	DefaultGRPCWebAddress = "0.0.0.0:9091"
+
+	// DefaultConcurrencyWorkers defines the default workers to use for concurrent transactions
+	DefaultConcurrencyWorkers = 20
+
+	// DefaultOccEanbled defines whether to use OCC for tx processing
+	DefaultOccEnabled = true
```

Comment:
shoudl we disable by default?

Reply:
This would mean that the default behavior becomes synchronous, which i guess is fine to be more cautious about